mhkit/tests/test_power.py

```diff
@@ -15,6 +15,12 @@ def setUpClass(self):
         self.samples = np.linspace(0, self.t, int(fs*self.t), endpoint=False)
```

Comment:
What is fs?

Reply:
@ssolson  fs is the sample frequency. If this were real data with time stamps it would be 1000 hz data.